ui/components/component-library/header-base/header-base.tsx

```diff
@@ -98,7 +96,6 @@ export const HeaderBase: React.FC<HeaderBaseProps> = ({
         <Box
           display={DISPLAY.FLEX}
           justifyContent={JustifyContent.flexEnd}
-          className="mm-header-base__end-accessory"
           ref={endAccessoryRef}
           style={
             children
```

Comment:
Removing unused classNames if engineers need a class they can assign one using the childrenWrapperProps